packages/editor/src/services/EditorStore.ts

```diff
@@ -107,6 +111,14 @@ export class EditorStore {
     return this.currentComponentsVersion === this.lastSavedComponentsVersion;
   }
 
+  get leftTabIdx() {
+    return this._leftTabIdx;
```

Comment:
No need to duplicate. `hoverComponentId` and other two property is duplicated for special reasons.